docs/content/_index.md

```diff
@@ -41,256 +42,96 @@ Using an innovative and re-architected distributed storage layer, YugabyteDB ser
-    linkUrl2="/preview/explore/fault-tolerance/"
-    linkText3="Effortless horizontal scaling"
-    linkUrl3="/preview/explore/linear-scalability/"
-  >}}
+##### Video: [Introducing YugabyteDB: The Distributed SQL Database for Mission-Critical Applications](https://www.youtube.com/watch?v=j24p07Frw00)
 
-{{< /sections/3-boxes >}}
+Learn about YugabyteDB and how it supports mission-critical applications.
 
-### Products
+##### Article: [How to Scale a Single-Server Database: A Guide to Distributed PostgreSQL](https://www.yugabyte.com/postgresql/distributed-postgresql/)
 
-{{< sections/3-boxes >}}
-  {{< sections/3-box-card
-    title="YugabyteDB"
-    description="The open source cloud-native distributed SQL database."
-    buttonText="Explore YugabyteDB"
-    buttonUrl="/preview/"
-    imageAlt="YugabyteDB Core"
-    imageUrl="/icons/database.svg"
-  >}}
+Why you need Distributed PostgreSQL to truly scale.
 
-  {{< sections/3-box-card
-    title="YugabyteDB Anywhere"
-    description="Deploy YugabyteDB across any cloud and manage deployments via automation."
-    buttonText="Documentation"
-    buttonUrl="/preview/yugabyte-platform/"
-    imageAlt="YugabyteDB Anywhere"
-    imageUrl="/icons/server.svg"
-  >}}
+##### Blog: [Data Replication in YugabyteDB](https://www.yugabyte.com/blog/data-replication/)
 
-  {{< sections/3-box-card
-    title="YugabyteDB Managed"
-    description="Create and connect to a scalable, resilient, PostgreSQL-compatible database in minutes."
-    buttonText="Documentation"
-    buttonUrl="/preview/yugabyte-cloud/"
-    imageAlt="YugabyteDB Managed"
-    imageUrl="/icons/cloud.svg"
-  >}}
+Learn the different data replication options available with YugabyteDB.
 
-  {{< sections/3-box-card
-    title="YugabyteDB Voyager"
-    description="Simplify migration from legacy and cloud databases to YugabyteDB."
-    buttonText="Documentation"
-    buttonUrl="/preview/yugabyte-voyager/"
-    imageAlt="YugabyteDB Managed"
-    imageUrl="/images/migrate/migration-icon.svg"
-  >}}
-{{< /sections/3-boxes >}}
+##### Documentation: [YugabyteDB architecture](preview/architecture/)
 
-<!--
-### New to YugabyteDB?
+Learn about the internals of query, transactions, sharding, replication, and storage layers.
 
-{{< sections/3-boxes >}}
-  {{< sections/3-box-card
-    title="Key benefits"
-    description="What is YugabyteDB, and why it should power your cloud-native applications."
-    buttonText="Key benefits"
-    buttonUrl="/preview/features/"
-  >}}
+##### Blog: [Improving PostgreSQL: How to Overcome the Tough Challenges with YugabyteDB](https://www.yugabyte.com/blog/improve-postgresql/)
 
-  {{< sections/3-box-card
-    title="Quick start"
-    description="Create your first cluster, explore distributed SQL, and build a sample application in 15 minutes."
-    buttonText="Get Started"
-    buttonUrl="/preview/quick-start-yugabytedb-managed/"
-  >}}
+Problem areas in PostgreSQL and how to resolve them in YugabyteDB.
```

Comment:
You can probably add this blog as it is has spring boot content - https://www.yugabyte.com/blog/retry-mechansim-spring-boot-app/